npm/vue/src/index.ts

```diff
@@ -1,12 +1,17 @@
+/* eslint-disable no-redeclare */
 /// <reference types="cypress" />
-import { Component, ComponentPublicInstance } from 'vue'
+import { ComponentPublicInstance, VNodeProps, AllowedComponentProps,
+  ComponentCustomProps, ExtractPropTypes, ExtractDefaultPropTypes,
+  Component, DefineComponent, FunctionalComponent, ComputedOptions,
+  MethodOptions, ComponentOptionsMixin, EmitsOptions, ComponentOptionsWithObjectProps, ComponentPropsOptions, ComponentOptionsWithArrayProps, ComponentOptionsWithoutProps } from 'vue'
 import { MountingOptions, VueWrapper, mount as VTUmount } from '@vue/test-utils'
 import {
   injectStylesBeforeElement,
   StyleOptions,
   ROOT_ID,
   setupHooks,
 } from '@cypress/mount-utils'
+import { GlobalMountOptions } from '@vue/test-utils/dist/types'
```

Comment:
Fine for now but we should export this from the root of VTU

Reply:
I found another way ! No need to export anything to keep the types tight.